app/src/ble.c

```diff
@@ -63,12 +65,14 @@ enum advertising_type {
 
 #define ZMK_ADV_CONN_NAME                                                                          \
     BT_LE_ADV_PARAM(BT_LE_ADV_OPT_CONNECTABLE | BT_LE_ADV_OPT_ONE_TIME, BT_GAP_ADV_FAST_INT_MIN_2, \
-                    BT_GAP_ADV_FAST_INT_MAX_2, NULL)
+                    BT_GAP_ADV_FAST_INT_MAX_2, NULL) // 100 - 150 ms advertising interval
 
+/* How are static variables stored across boots? */
 static struct zmk_ble_profile profiles[ZMK_BLE_PROFILE_COUNT];
 static uint8_t active_profile;
+bool active_profile_seeking_connection = true;
 
-#define DEVICE_NAME CONFIG_BT_DEVICE_NAME
+#define DEVICE_NAME CONFIG_BT_DEVICE_NAME // where is this defined?
```

Comment:
anything prefixed with CONFIG_ gets set in one of the KConfig files, so adv360_*side*_defconfig as well as a few other files in the adv360 directory, a lot of parameters are set at compile time by the kconfig file in the root of app as well